docs/ManualClusterSetup.md

```diff
@@ -7,17 +7,19 @@ This document describes requirements and best practices for setting up a cluster
 2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
 
 They must have the following software installed.
- * Docker v1.4 or greater: [Installing Docker](https://docs.docker.com/installation/)
+ * Docker v1.8 or greater: [Experimental Docker](https://experimental.docker.com)
  * etcd installed and available on each node: [etcd Documentation](https://coreos.com/etcd/docs/2.0.8/)
  * `ipset`, `iptables`, and `ip6tables` kernel modules.
 
 ## Best Practices
 
 ### CoreOS
-If CoreOS is your distribution, we recommend using cloud-config to bootstrap your etcd cluster.  [CoreOS Cluster Discovery](https://coreos.com/docs/cluster-management/setup/cluster-discovery/) is a good place to start.
+A prepackaged version of CoreOS with the most recent version of Docker is not yet available.  Calico uses the [libnetwork plugin](https://github.com/docker/libnetwork) which is currently only available in Experimental Docker v1.8.
```

Comment:
maybe say required version rather than most recent (I'd argue that it is available with the most recent released version which is generally what people would regard as the more recent...)